packages/react/src/primitives/Breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import { classNameModifierByFlag } from '../shared/utils';
+import {
+  BreadcrumbsLinkProps,
+  ForwardRefPrimitive,
+  Primitive,
+  BaseBreadcrumbLinkProps,
+} from '../types';
+import { ComponentClassNames } from '../shared/constants';
+import { Link } from '../Link';
+import { Text } from '../Text';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbLinkPrimitive: Primitive<BreadcrumbsLinkProps, 'a'> = (
+  { className, children, href, ...rest },
+  ref
+) => {
+  const { isCurrent } = React.useContext(BreadcrumbsContext);
```

Comment:
With `isCurrent` resolving to the same value for each ` BreadcrumbLinkPrimitive`, it's a bit unclear to me how the conditional logic works if more than one `BreadcrumbLinkPrimitive` is passed as the `children` to `Breadcrumbs`?

Reply:
If you have 2 BreadcrumbLinks inside 1 BreadcrumbItem, they would both get the same `isCurrent`. Although, hopefully you wouldn't do that